values.yaml

```diff
@@ -297,7 +297,6 @@ nginx:
 mongodb:
   enabled: false
   architecture: "replicaset"
-  replicaCount: 3
```

Comment:
Kind of unrelated, figured it would be better to default to the upstream default here.